sdk_container/src/third_party/coreos-overlay/coreos-base/oem-packet/oem-packet-0.2.2-r2.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 Kinvolk GmbH. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Equinix Metal suite for Equinix Metal"
+HOMEPAGE="https://www.equinix.com/"
+SRC_URI=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 arm64"
+IUSE=""
+
+OEM_NAME="Packet"
```

Comment:
We can rename this variable, it's used in the shell prompt message but otherwise I think it's not relevant